sdk/include/psdk/msxml2.idl

```diff
@@ -1258,7 +1258,7 @@ interface IXSLProcessor : IDispatch
     [id(DISPID_XMLDOM_PROCESSOR_SETSTARTMODE)]
     HRESULT setStartMode(
         [in] BSTR p, 
-        [in, defaultvalue(L"")] BSTR uri);
+        [in] BSTR uri);
```

Comment:
This is likely to be synced code with Wine. Upstreaming should be done.

Reply:
So, yeah, they will tell you that they did this also to be compatible with the headers/IDLs of MS PSDK. Note also that when using MIDL this will generate a prototype with something like "_In_ BSTR uri = L"" .

Reply:
But the wine code already differs here: https://github.com/wine-mirror/wine/blob/90a1e5d943724f8cb5e51fe806cb38ce6c98c4c1/include/msxml2.idl#L1259 (it uses "" instead of L"")

Reply:
That's handled by widl and doesn't affect the warning. I will simply selectively add "-Wno-write-strings" where needed. It's not optimal, but...

Reply:
I'm in favor of removing the default value. `L""` is not a valid BSTR anyway so any code that uses it would be broken.

Reply:
Works for me. I'll add the usual ```#ifdef __REACTOS__``` so that everyone is hopefully happy.